### ISSUE TITLE
Dockable Viewer List

### ISSUE THREAD

**Issue by RancidSwine** [Chatterino/chatterino2#1273]
**Is your feature request related to a problem? Please describe.**

I've been using Chatterino for a long time now and recently recommended it to a streamer. They like it a lot, the only thing they'd want to be able to do is dock the viewer list next to the chat room in a tab. The streamer likes to frequently check which mods are online, so he'd love to be able to just switch from his chat tab to a viewer list tab, or maybe even create a split with the chat/viewer list.

**Describe the solution you'd like**

I'd want the ability to add the viewer list as a tab, just like a chat room, whispers or mentions.

**Describe alternatives you've considered**

**Additional context**

**Comment by RAnders00** [comment]
You can already sort of achieve this by opening another (e.g. empty) split, then putting the viewer list over it:

![image](https://user-images.githubusercontent.com/1629196/64494642-d5c31900-d28f-11e9-835f-49408fa4cf08.png)

Might only work with "Always on top" setting enabled though.

**Comment by SoWhoYou** [comment]
This would be the only think that I really think is missing, would love to see this added!

**Comment by RancidSwine** [comment]
> 
> 
> You can already sort of achieve this by opening another (e.g. empty) split, then putting the viewer list over it:
> 
> ![image](https://user-images.githubusercontent.com/1629196/64494642-d5c31900-d28f-11e9-835f-49408fa4cf08.png)
> 
> Might only work with "Always on top" setting enabled though.

That's actually not a bad idea. Like I said in my post, this is something a streamer I watch wanted to be able to do. He would only use Chatterino for his own chat while streaming, and since he's using Chatty right now he's used to just always seeing his viewer list. I guess he wouldn't want to set this up every single time he opens Chatterino. 
I don't know how difficult it'd be to add this or how many people would even want this, but maybe this can be added as an option? Like an "always show viewer list next to chat" option that looks sorta like in Chatty:

![](https://chatty.github.io/img/dark.png)

**Comment by TheRealDadbeard** [comment]
Yes please. The only thing really that I miss from Chatty

**Comment by TeemuVi** [comment]
Yes it would be great if I could open the viewer list as a half sized chat box

**Comment by Luna-Nocturna** [comment]
Google showed this as the very first result and i couldn't find much else. Is this still not a thing? Was trying to move from Chatty to Chatterino but this is kind of a deal breaker for me.

**Comment by Felanbird** [comment]
> Google showed this as the very first result and i couldn't find much else. Is this still not a thing? Was trying to move from Chatty to Chatterino but this is kind of a deal breaker for me.

I don't believe anyone was really interested in adding such a feature, and if there was any interest hidden somewhere, it likely would have died when Twitch took the viewer list away from anyone who isn't a mod(and broadcaster), as in my personal opinion, mod+ only features don't tend to get that much interest, unless they actually take 2 seconds to implement, like `/warn`.

Also speaking from personal experience, if I was watching a streamer and they talked @ me because I appeared in the chatter list, not because I spoke in chat, I'd close the stream so fast.

**Comment by Luna-Nocturna** [comment]
> > Google showed this as the very first result and i couldn't find much else. Is this still not a thing? Was trying to move from Chatty to Chatterino but this is kind of a deal breaker for me.
> 
> I don't believe anyone was really interested in adding such a feature, and if there was any interest hidden somewhere, it likely would have died when Twitch took the viewer list away from anyone who isn't a mod(and broadcaster), as in my personal opinion, mod+ only features don't tend to get that much interest, unless they actually take 2 seconds to implement, like `/warn`.
> 
> Also speaking from personal experience, if I was watching a streamer and they talked @ me because I appeared in the chatter list, not because I spoke in chat, I'd close the stream so fast.

"Anyone who isn't a mod" - I get that, but I'm specifically looking around for an app from a mod/streamer perspective, not a regular chatter.

That said though, I don't get what you imply by it taken away? Even on channels I don't have mod privileges I can see the userlist just fine. (Even without the use of 3rd party.)

**Comment by Mm2PL** [comment]
W dniu 2.03.2025 o 08:51, Luna Nocturna pisze:
> (Even without the use of 3rd party.)
That's the thing. You can see (a part of) the chatter list in the first 
party chat client. But they don't allow third parties to access it 
without broadcaster permission. IIRC in channels with more than a 
hundred or so chatters the official thing might not show everyone.
